services/core-api/app/api/mines/reports/models/mine_report.py

```diff
@@ -106,11 +106,15 @@ def mine_report_status_code(self):
 
     @hybrid_property
     def report_type(self):
-        return "PRR" if self.permit_condition_category_code else "CRR"
+        return "PRR" if self.permit_condition_category_code or self.mine_report_permit_requirement_id else "CRR"
 
     @hybrid_property
     def report_name(self):
-        return self.mine_report_definition_report_name if self.mine_report_definition_report_name else self.permit_condition_category_description
+        if self.mine_report_definition_report_name:
```

Comment:
the update to report_name here totally fixed the table view to show the report name in the column. It crashed the BE before making the change.